cgp/node.py

```diff
@@ -175,7 +175,7 @@ class OperatorNode(Node):
 
     def __init_subclass__(cls: Type["OperatorNode"]) -> None:
         super().__init_subclass__()
-        OperatorNode._extract_input_names_from_def_output(cls)
+        OperatorNode._extract_address_names_from_def_output(cls)
```

Comment:
i think this should still be called "input names", "address names" is not very intuitive is it?

Reply:
agree